docs/getting_started/projects.md

```diff
@@ -1,4 +1,176 @@
-# <img class="dcr-icon" src="/img/dcr-icons/Transactions.svg" /> Overview
+# Projects
 
 ---
 
+All projects listed on this page are currently in active development by Decred Developers.
+All projects are open source with their source code available on GitHub.
+New contributors are welcome to contribute to any of these projects.
+
+---
+
+## dcrd
+
+- [decred/dcrd](https://github.com/decred/dcrd) (Go)
+
+dcrd is a full node implementation of Decred written in Go (golang).
+
+It acts as a fully-validating chain daemon for the Decred cryptocurrency.
+dcrd maintains the entire past transactional ledger of Decred and allows relaying
+of transactions to other Decred nodes around the world.
+The software was originally forked from [btcd](https://github.com/btcsuite/btcd),
+which is a bitcoin full node implementation that is still under active development.
+
+---
+
+## dcrwallet
+
+- [decred/dcrwallet](https://github.com/decred/dcrwallet) (Go)
+
+dcrwallet is the daemon that handles Decred wallet functionality for a single user.
+It manages all of your accounts, addresses, and transactions;
```

Comment:
Maybe use commas instead of `;`s here? Reads fine actually with ; but trips me up slightly

Reply:
Yeah I thought the semi-colon use was strange too tbh. I copied it from https://docs.decred.org/wallets/cli/dcrwallet-setup/